tests/managed-service/conftest.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def get_consumer_clusters():
```

Comment:
If you want to get the consumer indexes, there is an existing function in the `config` that already does it: `get_consumer_indexes_list` https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/__init__.py#L231.